app/static/css/style.css

```diff
@@ -68,6 +68,7 @@ body {
   text-align: center;
   margin: 0 auto;
   padding-top: 55px; /* Account for menu bar plus some extra spacing */
+  padding-bottom: 40px; /* Account for status bar plus some extra spacing */
```

Comment:
Is this part of these changes? It doesn't seem related.

Reply:
Removed.